### PR TITLE
[spirv-tools,spirv-reflect,spirv-headers] update ports

### PR DESCRIPTION
**Pull request by mathisloge**
**Describe the pull request**

- #### What does your PR fix?

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
yes
- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Review by Adela0814 — APPROVED** [review]

File: ports/spirv-headers/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "spirv-headers",
-  "version-date": "2021-03-25",
+  "version-date": "2022-01-19",
```
Comment (strega-nil-ms) [inline ports/spirv-headers/vcpkg.json:3]: Should this be

```suggestion
  "version": "1.3.204.1",
```

File: ports/spirv-reflect/vcpkg.json [hunk 2]
```diff
@@ -1,8 +1,9 @@
 {
   "name": "spirv-reflect",
-  "version-date": "2021-12-31",
+  "version-date": "2022-02-03",
```
Comment (strega-nil-ms) [inline ports/spirv-reflect/vcpkg.json:3]: same question here

**Review by strega-nil-ms — COMMENTED** [review]
I need an answer to these questions first

**Comment by strega-nil-ms** [comment]
Thanks @mathisloge !

**Pull request merged**